### DISCUSSION TITLE
How can we be transparent with anti-harrassment policies?

### DISCUSSION THREAD

**Discussion by pagerwho** [dataplat/DataSaturdays#167] [Q&A]
How do we communicate the policies around harassment?
How can someone report harassment to organizers?
In the unlikely event that an issue comes up with an organizer how can someone escalate this for mediation?

**Comment by tameraclark** [comment]
> 
> 
> How do we communicate the policies around harassment?
> How can someone report harassment to organizers?
> In the unlikely event that an issue comes up with an organizer how can someone escalate this for mediation?

This year at my event I issued a policy. That was sent as a separate email to all attendees, speakers, and sponsors.  For sponsors, it was printed out and left at their tables.  The policy included how to report an incident. A room was reserved to handle any complaints and a male and female representative were designated to handle any complaints. It was easy.  

The current process is up to each and every event to handle on their own.  I have done a few sessions collecting feedback and working with a few other SQLSaturday organizers so we could work together and feel supported.

**Comment by Alex-Yates** [comment]
I would advocate for a standard conduct/anti-harrassment policy.

Any event that decides not to use it, or decides to use a different policy, should make their reasons for not following the standard policy very clear to all attendees.

**Comment by stephlocke** [comment]
When I set up satRdays, we started with the policy from RLadies https://knowledgebase.satrdays.org/diversity/

**Comment by tameraclark** [comment]
@Alex-Yates  The current set up has a policy, the policy is vague and is likely written to protect the overseeing umbrella.  We did have that option turned on.  I think it is still currently optional.  We wrote one that was based on several policies that were suggested. 

During the SQLSat sessions, I did in 2019 it was discovered that a lot of folks would like to see to promotion of positive behaviors not a list of things that are not tolerated. Also, the current setup gives you no feedback on how to set up a policy, plan, and what to do if something happens. By nature, they can't tell you that because then they could be liable.

**Comment by HugoKornelis** [comment]
I prefer events to be mostly independent, but still held to a common standard. To implement that, my proposal would be a baseline policy / guidelines that event organizers are free to use. They can use their own but it has to include or exceed all elements of the baseline. And this baseline policy would not be anti-harrassment only, it would also be very explicit on inclusion.

Enforcement / dealing with complaints should be done locally, by the event organizer. If an event doesn't enforce their policy, then the ultimate penalty would be to lose eligibility to call themselves a DataSaturdays event, and lose access to whatever resources DataSaturdays makes available to events.

**Comment by rabryst** [comment]
My 2 cents is that the Calgary Data User Group uses a policy derived from confcodeofconduct.com.

**Comment by amtwo** [comment]
Gah. I missed this issue when I created #47. 

In that issue, I list these requirements for an event Code of Conduct:

> * The event-specific supplement should include who to report violations to, and how/where to contact them. 
>     * Reporting/assistance should be a _direct, unambiguous_ path from the Code of Conduct. (ie, a monitored email address, a front desk person at an in-person event, etc)
>     * Provide guidance to event organizers on eliminating single points of failure for individuals seeking help in reporting problems. Event organizers are often busy & not able to provide a fast initial response
> * The event Code of which is featured prominently & is easy to find. When an attendee is "in crisis" as the victim of harassment, it should not be difficult to find the Code of Conduct (and hence the path to reporting/assistance)


I think it's important & fair to have a baseline code of conduct that all events are held to. Each event will need to have a supplement to include reporting details for their event. That supplement could also be used for other additions to the baseline Code of Conduct. However, if an event organizer disagrees with the baseline Code of Conduct, they would be free to host their event elsewhere without a Code of Conduct, or with a different one. Allowing any event to have any code of conduct they wish (including one that may be contrary to the community's baseline standards), isn't a scalable approach if we want to ensure folks are able to bring their whole self to the community.

[a comment by an automated account is not shown]
